index.js

```diff
@@ -1,17 +1,61 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [],
+    this.length = 0
+  }
 
-  add(item) {}
+  add(item) {
+      this.items.push(item);
+      this.items.sort((a, b) => a - b);
+      this.length ++;
+    
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (typeof this.items[pos] !== 'number'){
+      throw new Error('OutOfBounds');
+    } else {
+      return this.items[pos];
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.length === 0){
+      throw new Error('EmptySortedList');
+    } else {
+      return this.items[this.length - 1];
+    }
```

Comment:
good job.
Another way you could have done it would have been with Math.max() method